<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqXYChartDisplayPanel</class>
 <widget class="QWidget" name="pqXYChartDisplayPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="pqTreeWidget" name="CompositeIndex">
     <column>
      <property name="text">
       <string>Select Block</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Attribute Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="AttributeMode">
       <item>
        <property name="text">
         <string>Point Data</string>
        </property>
        <property name="icon">
         <iconset resource="../pqComponents.qrc">
          <normaloff>:/pqWidgets/Icons/pqPointData16.png</normaloff>:/pqWidgets/Icons/pqPointData16.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cell Data</string>
        </property>
        <property name="icon">
         <iconset resource="../pqComponents.qrc">
          <normaloff>:/pqWidgets/Icons/pqCellData16.png</normaloff>:/pqWidgets/Icons/pqCellData16.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vertex Data</string>
        </property>
        <property name="icon">
         <iconset resource="../pqComponents.qrc">
          <normaloff>:/pqWidgets/Icons/pqPointData16.png</normaloff>:/pqWidgets/Icons/pqPointData16.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Edge Data</string>
        </property>
        <property name="icon">
         <iconset resource="../pqComponents.qrc">
          <normaloff>:/pqWidgets/Icons/pqRamp24.png</normaloff>:/pqWidgets/Icons/pqRamp24.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Row Data</string>
        </property>
        <property name="icon">
         <iconset resource="../pqComponents.qrc">
          <normaloff>:/pqWidgets/Icons/pqSpreadsheet16.png</normaloff>:/pqWidgets/Icons/pqSpreadsheet16.png</iconset>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>X Axis Data</string>
     </property>
     <layout class="QGridLayout" name="_2">
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="UseArrayIndex">
        <property name="whatsThis">
         <string>Select this option to use the indexes of the y-axis array for the x-axis points.</string>
        </property>
        <property name="text">
         <string>Use Array Index From Y Axis Data</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="UseDataArray">
        <property name="whatsThis">
         <string>Select this option to use a data array to define the x-axis points.</string>
        </property>
        <property name="text">
         <string>Use Data Array</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="XAxisArray">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="whatsThis">
         <string>Choose the name of the x-axis data array.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="SeriesGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Line Series</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>6</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="5" column="1">
       <widget class="QComboBox" name="AxisList">
        <property name="whatsThis">
         <string>Select the chart axes for the line series.</string>
        </property>
        <item>
         <property name="text">
          <string>Bottom-Left</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bottom-Right</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Top-Right</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Top-Left</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="StyleList">
        <property name="whatsThis">
         <string>Select the line style for the series.</string>
        </property>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Solid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dash</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dot</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dash Dot</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dash Dot Dot</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="AxisListLabel">
        <property name="text">
         <string>Chart Axes</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="StyleListLabel">
        <property name="text">
         <string>Line Style</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="Thickness">
        <property name="whatsThis">
         <string>Enter the thickness for the line.</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ThicknessLabel">
        <property name="text">
         <string>Line Thickness</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="pqColorChooserButton" name="ColorButton">
        <property name="whatsThis">
         <string>Press this button to choose a new color for the line.</string>
        </property>
        <property name="text">
         <string>Choose Color...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Line Color</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="pqTreeView" name="SeriesList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis">
         <string>This displays the list of line series. You can use the list to edit the series options.</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="MarkerStyleListLabel">
        <property name="text">
         <string>Marker Style</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="MarkerStyleList">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cross</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Plus</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Square</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Circle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Diamond</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqTreeView</class>
   <extends>QTreeView</extends>
   <header>pqTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>pqTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>pqTreeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>pqColorChooserButton</class>
   <extends>QPushButton</extends>
   <header>pqColorChooserButton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>CompositeIndex</tabstop>
  <tabstop>AttributeMode</tabstop>
  <tabstop>UseArrayIndex</tabstop>
  <tabstop>UseDataArray</tabstop>
  <tabstop>XAxisArray</tabstop>
  <tabstop>SeriesList</tabstop>
  <tabstop>ColorButton</tabstop>
  <tabstop>Thickness</tabstop>
  <tabstop>StyleList</tabstop>
  <tabstop>MarkerStyleList</tabstop>
  <tabstop>AxisList</tabstop>
 </tabstops>
 <resources>
  <include location="../pqComponents.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>UseDataArray</sender>
   <signal>toggled(bool)</signal>
   <receiver>XAxisArray</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
